pkgs/tools/security/yubihsm-shell/default.nix

```diff
@@ -48,12 +48,17 @@ stdenv.mkDerivation rec {
     curl
     openssl
   ] ++ lib.optionals stdenv.isLinux [
-    pcsclite
+    pcsclite.dev
   ] ++ lib.optionals stdenv.isDarwin [
     darwin.apple_sdk.frameworks.PCSC
     libiconv
   ];
 
+  buildPhase = lib.optional stdenv.isLinux ''
+    NIX_CFLAGS_COMPILE="$(pkg-config --cflags libpcsclite) $NIX_CFLAGS_COMPILE"
+    runHook buildPhase
+  '';
```

Comment:
```suggestion
  buildPhase = lib.optionalString stdenv.isLinux ''
    runHook preBuild
    NIX_CFLAGS_COMPILE="$(pkg-config --cflags libpcsclite) $NIX_CFLAGS_COMPILE"
    runHook postBuild
  '';
```

Reply:
Or maybe we could simply use a `preBuild` step ?

Reply:
Good call - adjusted!